### DISCUSSION TITLE
VNEXT & Postgres Migration

### DISCUSSION THREAD

**Discussion by kusold** [sysadminsmedia/homebox#375] [Q&A] [answered]
I'm in the process of onboarding all of my items into homebox. Long-term, I'm interested in using the Postgres backend from https://github.com/sysadminsmedia/homebox/pull/291 .

Is there a migration path from sqlite to postgres, or does it require starting over from scratch?

If there isn't a migration path planned, I could be interested in working on one utilizing the API.

**Comment by tankerkiller125** [comment, marked as answer]
There is no migration path as it stands do the fundamental differences between the DB systems. In theory one could export each table into a CSV using whatever tools they prefer, and then import the CSV into the tables on the other side. But at the moment that's the only solution I know of.

If you wanted to build something based on the API you should be able to do that.
